tests/UnitTests/BankTransactionUnitTest.php

```diff
@@ -30,4 +34,131 @@ public function testCanGetReferenceFromLine()
         $this->assertEquals("1", $reference->getLineId());
         $this->assertEquals("MEMO", $reference->getCode());
     }
+
+    public function testGetLineReturnsLinesInSameOrderButAlwaysWithTotalAtTheTop()
+    {
+        $total_line_count = 287;
```

Comment:
Isn't this a bit overkill?

Reply:
I did a little TDD to find out if this was actually what was causing the random order. So I wrote the test first and it didn't fail when I had `$total_line_count = 10`, only when I went up a bit higher did it start failing. So I figured I'd just do a whole lot. Perhaps it's a little overkill, but luckily unit tests are very fast.